lm_eval/base.py

```diff
@@ -397,6 +401,10 @@ def fewshot_examples(self, k, rnd):
 
         return rnd.sample(self._training_docs, k)
 
+    def doc_to_decontamination_query(self, doc):
+        print("Override doc_to_decontamination_query with document specific decontamination query.")
+        assert(False)
```

Comment:
Is there any reason that doc_to_decontamination_query is ever anything other than doc_to_text + doc_to_target? And why would we ever not be able to do decontamination for a task?

Reply:
See the individual tasks for the answer to the first question. Given it was not a one size fits all we don't want existing tasks to be broken, hence disabling by default.